bin/list

```diff
@@ -161,6 +161,34 @@ const renderDirectory = async dir => {
   return render(details)
 }
 
+
+
+// To enhance, one can take inspiration from https://github.com/jfhbrook/node-ecstatic/blob/fe17a5623c18319f82101b6549e9e2e526e03d1f/lib/ecstatic.js#L284
+const isNotModified = (req, etag, fileStats) => {
+  try {
```

Comment:
What do you expect to `throw` here? I find it best to put as little as possible in `try` blocks... like usually the one line that can throw. This makes it obvious to other developers where the problems lie.